src/site/_includes/components/CodelabsCallout.js

```diff
@@ -58,50 +78,12 @@ function CodelabCallout(slugs, lang) {
           </div>
           <nav class="callout__links" aria-label="Codelabs links">
             <ul role="list">
-              ${codelabs.map(codelab => html`
-                <li>
-                  <a href="${codelab.url}">
-                    <span>${md(codelab.data.title)}</span>
-                    <!-- icons/carat-forward.svg -->
-                    <svg width="24" height="24" viewBox="0 0 24 24" fill="currentColor" xmlns="http://www.w3.org/2000/svg"><path d="m8 7.34 4.58 4.365L8 16.07l1.41 1.34 6-5.705L9.41 6 8 7.34Z"/></svg>
-                  </a>
-                </li>
-              `).join('')}
+              ${codelabs.map(renderCodelab).join('')}
             </ul>
           </nav>
         </div>
       </aside>
-    `
-  }
-
-  /// DELETE THIS WHEN ROLLOUT COMPLETE
-  function renderCodelab(codelab) {
-    return html`
-      <li class="w-callout__listitem">
-        <a
-          class="w-callout__link w-callout__link--codelab"
-          href="${codelab.url}"
-        >
-          ${md(codelab.data.title)}
-        </a>
-      </li>
     `;
-  }
-
-  return html`
-    <div class="w-callout w-callout--2col">
-      <div class="w-callout__header">
-        <h2 class="w-callout__lockup w-callout__lockup--codelab">Codelabs</h2>
-        <div class="w-callout__headline">See it in action</div>
-        <div class="w-callout__blurb">
-          Learn more and put this guide into action.
-        </div>
-      </div>
-      <ul class="w-unstyled-list w-callout__list">
-        ${codelabs.map(renderCodelab)}
-      </ul>
-    </div>
-  `;
 }
 
-module.exports = CodelabCallout;
+module.exports = {CodelabsCallout, renderCodelab};
```

Comment:
renderCodelab is not a component, and should not be exported here in the coponents folder (idea is to have 1 dir = 1 component). Looks like it's just a piece of markup - how about moving it into the template instead of shortcode?